pkg/security/security_profile/dump/activity_dump.go

```diff
@@ -884,3 +888,64 @@ func (ad *ActivityDump) DecodeJSON(reader io.Reader) error {
 
 	return nil
 }
+
+// LoadActivityDumpsFromFiles load ads from a file or a directory
+func LoadActivityDumpsFromFiles(path string) (interface{}, error) {
```

Comment:
Nit: IMHO it would be more clear to split this into two func:

- one for a single ad, returning a single ad
- another one to load all ad in a directory, returning a list of ad (and calling the first one to avoid code duplication)